src/tables/colordata.cpp

```diff
@@ -149,6 +149,8 @@ int LibRaw::adobe_coeff(unsigned make_idx, const char *t_model,
       { 9423,-2839,-1195,-4532,12377,2415,-483,1374,5276 } },
     { LIBRAW_CAMERAMAKER_Canon, "EOS R5", 0, 0,
       { 9766,-2953,-1254,-4276,12116,2433,-437,1336,5131 } },
+    { LIBRAW_CAMERAMAKER_Canon, "EOS R6 Mark II", 0, 0,
+      { 8293,-1611,-1132,-4759,12711,2275,-1013,2415,5509 } },
```

Comment:
I don't get these D65 coeffs w/ the DNG Converter... They look identical to R6 btw, but the sensor is different.

Reply:
Obviously, one cannot use `raw-identify` (for everything) on a camera that is not yet supported and LibRaw doesn't know about. 😉 

> Does it match the DNG Converter output?

It does not:
```
ColorMatrix2                    : 0.9539 -0.2795 -0.1224 -0.4175 1.1998 0.2458 -0.0465 0.1755 0.6048
CameraCalibration2              : 1.0016 0 0 0 1 0 0 0 1.0046
AnalogBalance                   : 1 1 1
```

Reply:
For an unsupported camera LibRaw may return a 'best match' (look into adobe_coeff() code) and this is EOS R6 camera (ta-damm).
It is worthless to duplicate this data in separate table entry.

Reply:
I see. So the coefficients in my previous comments probably belong to another older model (I got those with libraw 0.20.2 which doesn't know the original R6 yet). So IIUC the correct entry would be
```
    { LIBRAW_CAMERAMAKER_Canon, "EOS R6 Mark II", 0, 0,
      { 9539,-2795,-1224,-4175,11998,2458,-465,1755,6048 } },
```
Do I need to apply also the CameraCalibration2 coefficients somewhere?